tests/core/test_flare.py

```diff
@@ -170,3 +211,34 @@ def test_uri_password_regex(self, mock_config, mock_tempdir, mock_strftime):
             line,
             password_tests['uri_password_expected']
         )
+
+    @attr(requires='core_integration')
+    @mock.patch('utils.flare.strftime', side_effect=mocked_strftime)
+    @mock.patch('tempfile.gettempdir', side_effect=get_mocked_temp)
+    @mock.patch('utils.flare.get_config', side_effect=get_mocked_config_with_proxy)
+    def test_uri_set_proxy(self, mock_config, mock_tempdir, mock_strftime):
+        f = Flare()
+        request_options = {}
+        f.set_proxy(request_options)
+        expected = 'http://proxy_user:proxy_pass@proxy.host.com:3128'
+        self.assertEqual(expected, request_options['proxies']['https'])
+
+    @attr(requires='core_integration')
+    @mock.patch('utils.flare.strftime', side_effect=mocked_strftime)
+    @mock.patch('tempfile.gettempdir', side_effect=get_mocked_temp)
+    @mock.patch('utils.flare.get_config', side_effect=get_mocked_config_with_proxy)
+    def test_uri_no_verify_ssl(self, mock_config, mock_tempdir, mock_strftime):
+        f = Flare()
+        request_options = {}
+        f.set_ssl_validation(request_options)
+        self.assertFalse(request_options['verify'])
+
+    @attr(requires='core_integration')
+    @mock.patch('utils.flare.strftime', side_effect=mocked_strftime)
+    @mock.patch('tempfile.gettempdir', side_effect=get_mocked_temp)
+    @mock.patch('utils.flare.get_config', side_effect=get_mocked_config)
+    def test_uri_verify_ssl_default(self, mock_config, mock_tempdir, mock_strftime):
+        f = Flare()
+        request_options = {}
+        f.set_ssl_validation(request_options)
+        self.assertIsNone(request_options.get('verify'))
```

Comment:
Unfortunately `assertIsNone` isn't supported by python 2.6, and we want to support 2.6...

Could you change this to `self.assertEquals(..., None)`?